inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/flink/FlinkOperation.java

```diff
@@ -161,15 +174,46 @@ public void genPath(FlinkInfo flinkInfo, String dataflow) throws Exception {
         flinkInfo.setLocalJarPath(jarPath);
         log.info("get sort jar path success, path: {}", jarPath);
 
+        List<String> nodeTypes = new ArrayList<>();
+        if (StringUtils.isNotEmpty(dataflow)) {
+            JsonNode streams = JsonUtils.parseTree(dataflow).get(InlongConstants.STREAMS);
+            for (int i = 0; i < streams.size(); i++) {
+                JsonNode relations = streams.get(i).get(InlongConstants.RELATIONS);
+                for (int j = 0; j < relations.size(); j++) {
+                    String inputNames = OBJECT_MAPPER.convertValue(relations.get(j).get(InlongConstants.INPUTS),
+                                    new TypeReference<List<String>>(){}).stream().findFirst().orElse(null);
+                    String outputNames = OBJECT_MAPPER.convertValue(relations.get(j).get(InlongConstants.OUTPUTS),
+                            new TypeReference<List<String>>(){}).stream().findFirst().orElse(null);
+
+                    if (inputNames.equals(outputNames)) {
+                        String message = String.format("input nodeName: %s equals to output nodeName: %s",
```

Comment:
Suggest more clearly, for example, "input nodeName %s cannot be equal to output nodeName %s".

Reply:
ok